apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -75,10 +93,16 @@ const useSlots = ({
     },
     {
       enabled: !!startTime && !!endTime && enabled,
-      refetchInterval: 3000,
+      refetchInterval,
       trpc: { context: { skipBatch: true } },
     }
   );
+  useEffect(() => {
+    if (!!data && fetchStatus === "idle") {
```

Comment:
refetchCount should be it's own hook, otherwise other re-renders may interfere here

Also we probably want to useRef() this counter, no need to rerender 🙏